### DISCUSSION TITLE
what it means

### DISCUSSION THREAD

**Discussion by 7by24** [Chia-Network/chia-blockchain#5885] [Support]
upgrade to 1.1.6/1.1.7.dev0
and this warning message showing up.
is there anybody who experienced this before?
```
full_node full_node_server: WARNING  Invalid handshake with peer. Maybe the peer is running old software
```

**Comment by katmai** [comment]
it says in the message: the peer is running old software.

**Reply by 7by24** [reply]
I am wondering who's peers? mine or any others?

**Reply by tiny0little** [reply]
`peer` mean other side

**Comment by BrandtH22** [comment]
@ihola777 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
